docs/pymupdf-pro.rst

```diff
@@ -0,0 +1,107 @@
+
+.. include:: header.rst
+
+
+
+.. _pymupdf-pro
+
+PyMuPDF Pro
+=============
+
+
+Enhance |PyMuPDF| capability with **Office** document support.
+
+|PyMuPDF Pro| offers all the features of |PyMuPDF|, plus enhanced functionality to support **Office** documents.
+
+- Load, parse and extract text data from **Office** files
+- Able to render **Office** files
+
+
+Office file support
+----------------------
+
+In addition to the `standard file types supported by PyMuPDF <Supported_File_Types>`, |PyMuPDF Pro| supports:
+
+- DOC/DOCX
+- PPT/PPTX
+- XLS/XLXS
+- HWP/HWPX
+
+
+Usage
+--------------
+
+Installation
+~~~~~~~~~~~~~~~~~~
+
+Install via pip with:
+
+.. code-block:: bash
+
+    pip install pymupdfpro
+
+
+Loading an **Office** document
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Import |PyMuPDF Pro| and you can then reference **Office** documents directly, e.g.:
+
+.. code-block:: python
+
+    import pymupdfpro
+    doc = pymupdf.open("my-office-doc.xls")
+
+.. note::
+
+    All standard |PyMuPDF| functionality is exposed as expected - |PyMuPDF Pro| handles the extended **Office** file types
+
+
+From then on you can work with document pages just as you would do normally, but with respect to the `restrictions <PyMuPDFPro_Restrictions>`.
+
+
+.. _PyMuPDFPro_Restrictions:
+
+Restrictions
+~~~~~~~~~~~~~~~~~~~~
+
+
+|PyMuPDF Pro| functionality is restricted without a license key as follows:
+
+    **Only the first 10KB of any document will be available as extracted text.**
```

Comment:
Should read:
**Only the content of the first three pages of any document will be available.**